### ISSUE TITLE
IRRemote stopped working after update to 5.8.0

### ISSUE THREAD

**Issue by bluebird11** [arendst/Sonoff-Tasmota#993]
It used to work before, now I get:

```
 11:07:11 CMD: IRsend {"protocol":"NEC" ,"bits":32 ,"data":1286815935}
 11:07:11 MQT: stat/sonoff-livingroom-light/RESULT = {"IRSend":"NoPROTOCOL, BITS or DATA"}
```

No matter what command I send.

I have IRremoteESP8266 1.2.0 installed, the code doesn't compile with higher versions.

Did something change?

**Comment by arendst** [comment]
Yes, something did change.

As a workaround please use uppercase words for PROTOCOL, BITS and DATA and report back if that solves the issue.

**Comment by bluebird11** [comment]
Yes, that fixed it, thanks a lot and thank you very much for your work and the updates, SetOption13 is awesome!

Can you also consider adding support for 5 relays (main relay + 4 GPIO)? :)

**Comment by arendst** [comment]
Implemented in master release v5.9.0
